src/lib/helpers.ts

```diff
@@ -249,3 +251,21 @@ export const CatchCollectionNotFoundException = (error) => {
 
   throw error
 }
+
+/**
+ * Converts all object keys to snake case.
+ * @param object — The object to convert.
+ * @return — Returns the object with converted keys.
+ */
+export const snakeCaseKeys = (
+  obj: Record<string, any>
+): Record<string, any> => {
+  return mapKeys(obj, (_, key) => {
+    // Special case for ID to not be converted to I_D
+    if (key.includes("ID")) {
```

Comment:
good that we have no "ID" field on it's own.

Reply:
Which would then be converted to `id` by this function.